internal/multigitter/review.go

```diff
@@ -0,0 +1,253 @@
+package multigitter
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/lindell/multi-gitter/internal/scm"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// Reviewer reviews pull requests in an organization
+type Reviewer struct {
+	VersionController VersionController
+	FeatureBranch     string
+	Comment           string
+	All               bool
+	Batch             string
+	Pager             string
+	DisablePaging     bool
+	IncludeApproved   bool
+}
+
+const (
+	header        = "# *********************************************************"
+	reviewOptions = "[a/r/c/n/d/h (help)]"
+	batchApprove  = "approve"
+	batchReject   = "reject"
+	batchComment  = "comment"
+)
+
+type approvdPr struct {
+	scm.PullRequest
+	approved bool
+}
+
+// Reviewer reviews pull requests in an organization
+func (s Reviewer) Review(ctx context.Context) error {
+	prs, err := s.VersionController.GetPullRequests(ctx, s.FeatureBranch)
+	if err != nil {
+		return err
+	}
+
+	var approvedPrs []approvdPr
+
+	for _, pr := range prs {
+		approved, err := s.VersionController.IsPullRequestApprovedByMe(ctx, pr)
```

Comment:
```suggestion
	for _, pr := range prs {
		log := log.WithField("pr", pr.String())
		approved, err := s.VersionController.IsPullRequestApprovedByMe(ctx, pr)
```

Reply:
To be able to get information of which PR went wrong when logging, we need to include the PR as a part of the logging.